apps/whale-api/src/module.api/stats.controller.ts

```diff
@@ -96,6 +109,16 @@ export class StatsController {
     }
   }
 
+  @Get('/rewards/distribution')
+  async getRewardDistribution (
+    @Query('height', ParseIntPipe) reductionHeight = 0
+  ): Promise<RewardDistributionData> {
+    const height = this.options.eunosHeight + reductionHeight * this.options.emissionReductionInterval
+    const subsidy = this.blockSubsidy.getBlockSubsidy(height)
```

Comment:
Since /stats/ is live data, we pass in the latest height from BlockMapper instead of computing the reduction intervals.

```suggestion
  async getRewardDistribution (): Promise<RewardDistributionData> {
    const block = requireValue(await this.blockMapper.getHighest(), 'block')
    const subsidy = this.blockSubsidy.getBlockSubsidy(block.height)
```